### DISCUSSION TITLE
Is it possible to add a transition to the Navbar in AppShell?

### DISCUSSION THREAD

**Discussion by itmecho** [mantinedev/mantine#1027] [Q&A] [answered]
I'd like to add a slide in/out transition to the navbar to make it a little smoother but I can't seem to figure it out! Has anyone had any luck with this?

**Comment by rtivital** [comment]
If you need animations, use Drawer component

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/boring-hawking-3o2ovb?file=/src/App.tsx

**Comment by siba2893** [comment]
Hey rtivital the example you proposed doesn't work right now. The codebox is broken.

**Reply by rtivital** [reply]
It works fine for me

**Reply by IDrumsey** [reply]
@siba2893 Try zooming out in the browser.

**Reply by primeviltom** [reply]
I'm trying to achieve the same thing as OP. In the codesandbox example, the Navbar is always displayed when the breakpoint is at a smaller value, regardless of clicking the burger. The width prop of the Navbar appears to be overridden in this case.

Is it possible to achieve the animated behaviour at a smaller breakpoint? 

Edit: I think a common use case would be to have an always-displayed Navbar with larger screens, and an animated slide-in Navbar + Burger for smaller screens as part of the AppShell.
